src/codeu/chat/server/View.java

```diff
@@ -78,6 +78,81 @@ public View(Model model) {
   @Override
   public Message findMessage(Uuid id) { return model.messageById().first(id); }
 
+  @Override
+  public Collection<String> userStatusUpdate(String name, Uuid owner) {
+
+    Collection<ConversationPayload> allConversations = all(model.conversationPayloadById());
+    Collection<String> contributions = new ArrayList<String>();
+
+    final User foundOwner = model.userById().first(owner);
+    final User foundUser = model.userByText().first(name);
+
+    if(foundUser != null) {
+      if(foundOwner.UserSet.contains(foundUser.id)) {
```

Comment:
IIUC, you accept the UUID from the incoming message (correct me if I'm wrong here). What will happen if the incoming message contains a garbage UUID? I understand the current client will not send it, but what if someone writes a client which does?

This is not a big concern for the scope of this project, but definitely a thing to keep in mind. It is better to at least throw an exception or error for such cases rather than crashing with a RuntimeException (like NullPointerException).